code/modules/mob/living/carbon/human/human_abilities.dm

```diff
@@ -170,7 +172,17 @@ CULT
 
 	update_button_icon()
 
-	addtimer(CALLBACK(src, PROC_REF(update_button_icon)), amount)
+	addtimer(CALLBACK(src, PROC_REF(on_cooldown_over)), amount, TIMER_UNIQUE|TIMER_STOPPABLE)
+
+/datum/action/human_action/activable/proc/on_cooldown_over()
+	if(!owner)
+		return
+	update_button_icon()
+	if(cooldown_message_on)
+		if(isnull(cooldown_message))
+			to_chat(owner, SPAN_XENODANGER("You feel your strength return! You can use [name] again!"))
+		else
```

Comment:
i don't quite get why this is like this, if it's just copypasta let me know but uhhh... we have TIMER_COOLDOWN_START and stuff

Reply:
I really just looked at xeno actions and looked at human_abilities... yeha